Polyhedron/doc/Polyhedron/CGAL/Polyhedron_3.h

```diff
@@ -1594,7 +1594,27 @@ class Polyhedron_3 {
   */ 
   void delegate( CGAL::Modifier_base<HDS>& m); 
 
+  
   /// @}
 
 }; /* end Polyhedron_3 */
+
+
+/*!
+\relates Polyhedron_3
+\ingroup PkgPolyhedronIOFunc
+
+*/
+template <class PolyhedronTraits_3>
+bool read_off( std::istream& in, CGAL::Polyhedron_3<PolyhedronTraits_3>& P);
+
+/*!
+\relates Polyhedron_3
+\ingroup PkgPolyhedronIOFunc
+
+*/
+template <class PolyhedronTraits_3>
+bool write_off( std::ostream& out, CGAL::Polyhedron_3<PolyhedronTraits_3>& P);
```

Comment:
`read_off` and `write_off` are added to the documentation of Polyhedron_3 without any documentation. I suggest to add a link to the relevant part of the concept, if there is one, or to say that it is equivalent to `in >> P` and `out << P`.